taca/analysis/analysis.py

```diff
@@ -1,19 +1,20 @@
 """ Analysis methods for TACA """
 import csv
+from datetime import datetime
```

Comment:
hmm is not very clear to me what [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) says about this :-/

It says standard imports first, but that's what I do as well, is just that I separate the total and partial imports..